src/state/image-dataset/csv-dataset/test/CsvRequest.test.ts

```diff
@@ -335,4 +337,109 @@ describe("CsvRequest", () => {
         expect(await csvData.getFileInfoByCellId("3")).to.not.be.undefined;
         expect(await csvData.getFileInfoByCellId("5")).to.be.undefined;
     });
+
+    describe("retrieves Vol-E query parameters", () => {
+        const CELL_1_PARAMS =
+            "interp=0&view=Z&slice=0.5,0.5,0.75&bright=30&dens=60&c0=sen:1,isv:150&c2=ven:1,clz:1,lut:v15:v230";
+        const CELL_2_PARAMS =
+            "url=https://example1.com,https://example2.com&t=15&c0=ven:0&c1=ven:1";
+
+        async function validateCell1Params(csvDataset: CsvRequest): Promise<void> {
+            // Validate cell 1
+            const cell1Info = await csvDataset.getFileInfoByCellId("cell1");
+
+            const cell1VoleArgs = cell1Info?.voleUrlParams?.args;
+            const cell1VoleSettings = cell1Info?.voleUrlParams?.viewerSettings;
+            expect(cell1VoleArgs).to.not.equal(undefined);
+            expect(cell1VoleSettings).to.not.equal(undefined);
+
+            // Viewer settings
+            expect(cell1VoleSettings?.interpolationEnabled).to.equal(false);
+            expect(cell1VoleSettings?.viewMode).to.equal(ViewMode.xy);
+            expect(cell1VoleSettings?.slice).to.deep.equal({ x: 0.5, y: 0.5, z: 0.75 });
+            expect(cell1VoleSettings?.brightness).to.equal(30);
+            expect(cell1VoleSettings?.density).to.equal(60);
+
+            // Channel data
+            const cell1Channels = cell1VoleArgs?.viewerChannelSettings?.groups[0].channels;
+            assert(cell1Channels !== undefined);
+            expect(cell1Channels[0].match).to.equal(0);
+            expect(cell1Channels[0].enabled).toBeFalsy();
```

Comment:
nit: should we standardize the assertions on one style? like using `to.be.false` here?